entrypoint/main.go

```diff
@@ -120,9 +121,13 @@ func entrypoint() error {
 				return errSt
 			}
 		}
+
+		if len(command) > 1 && command[0] == "icinga2" && command[1] == "daemon" {
```

Comment:
Why this specific check? Isn't entrypoint run for PID 1 inside the container anyways, so if you run `docker exec`, we won't get here? Or am I missing some other case?

Reply:
Why PID==1 or why icinga2 daemon?

Reply:
Actually both. Shouldn't Docker only call the entrypoint if it is PID 1? And would it hurt to start other commands with dumb-init?

Reply:
1. Just to be sure. Not PID 1 – no need for this.
2. Yes, e.g. bash due to dumb-init **-c**.

Reply:
What's the problem with `dumb-init -c bash`?

Reply:
It's unnecessary.

Reply:
But if it works fine (which I think it does), this special handling here is unnecessary. ;)

Reply:
I consider `dumb-init -c bash` more unnecessary.

Reply:
I don't see why we would want special handling here. There are a few situations where this might give unexpected behavior:

1. Explicitly calling `/usr/sbin/icinga2` instead of just `icinga2`
2. Running the container with a command like `/data/init.sh` that will later `exec icinga2`
3. Possibly one could pass arguments between `icinga2` and `daemon`? At least it does not complain if you run something like `icinga -Dfoo=bar daemon`.